src/main/content/guides.html

```diff
@@ -1,29 +1,14 @@
 ---
-layout: default
+layout: guides
 title: Guides
 css:
 - guide-card
 - guides
+- doc-header
 js: guides
 permalink: /guides/
 ---
 
-<!-- INTRO -->
-<div id="guides_intro_background" class="black_blue_gradient_background">
-    <div class="container">
-        <div class="row">
-            <div class="col-sm-4">
-                <img id="intro_logo" src="{{ "/img/guides_page_title_large.svg" | relative }}" class="img-responsive" alt="Guides">
```

Comment:
We need to delete `guides_page_title_large.svg` if we no longer need it.

Reply:
Done